isb_lib/opencontext_adapter.py

```diff
@@ -175,6 +175,18 @@ def reparse_as_core_record(thing: isb_lib.models.thing.Thing) -> typing.List[typ
         raise
 
 
+def identifier_from_thing_dict(thing_dict: typing.Dict) -> str:
+    opencontext_id = thing_dict["citation uri"]
+    # check the type here because some records don't have it and have a boolean False instead
+    if opencontext_id is not None and type(opencontext_id) is str:
+        opencontext_id = isb_lib.normalized_id(opencontext_id)
+    else:
+        opencontext_id = thing_dict["uri"]
```

Comment:
I really don't know what else to do here -- we are going to end up in a situation where the things live on the box but don't exist in solr or in iSC.  However, given the way things are set up, we do need this identifier to proceed.